web/docs/advanced/email/email.md

```diff
@@ -384,6 +384,10 @@ The `send` method accepts an object with the following fields:
 
     The email address of the sender.
 
+- `replyTo?: string`
+
+  The email address to which the recipient can reply to. Only available for SendGrid provider.
```

Comment:
I only checked the Mailgun provider which does not offer a `reply_to` option. I'm pretty sure, that it can be implemented for the SMTP provider as well. I will have a look at it.